packages/dates/src/date-time-utils/DateTimeUtils.ts

```diff
@@ -142,7 +142,7 @@ export function formatDate(
     return fallback;
   }
 
-  return date.toLocaleString(DATE_DISPLAY_VARIANTS[variant]);
+  return date.setLocale('en-US').toLocaleString(DATE_DISPLAY_VARIANTS[variant]);
```

Comment:
@umrzoq-toshkentov Will it solve the problem overall with the date locale? Maybe we can use `Settings.defaultLocale`. 
I do not remember what was done last time this problem appeared.  @doniyor2109 do you remember?

Reply:
@timurhamitov I think we did not solve it last time.

Reply:
@timurhamitov let me try it. If I find another solution, I will update PR

Reply:
I am also for `Settings.defaultLocale` as there could be other cases where we format date

Reply:
Updated to `defaultLocale`
cc @timurhamitov @doniyor2109